eng/Packages.Data.props

```diff
@@ -206,7 +206,7 @@
   </ItemGroup>
 
   <ItemGroup Condition="'$(IsGeneratorLibrary)' == 'true'">
-    <PackageReference Update="Microsoft.TypeSpec.Generator.ClientModel" Version="1.0.0-alpha.20250407.1" />
+    <PackageReference Update="Microsoft.TypeSpec.Generator.ClientModel" Version="1.0.0-alpha.20250417.1" />
```

Comment:
This version doesn't match the emitter version.

Reply:
We should add a CI check somewhere (maybe in Generate.ps1) to ensure they match.

Reply:
because I need the change from https://github.com/microsoft/typespec/pull/7010, released a newer version of generator

Reply:
This is temporary to make the CI pass, will update the latest version once https://github.com/microsoft/typespec/pull/7010 merged